### ISSUE TITLE
Hardcoded /sdcard/ breaks things

### ISSUE THREAD

**Issue by scottkennedy-ck** [facebook/screenshot-tests-for-android#27]
https://github.com/facebook/screenshot-tests-for-android/commit/05cb5ac8ed0f832dbf10ba78a451013154f68316 broke this for me.

From a stock emulator running API 22:
```
root@generic_x86_64:/ # cd /sdcard/
/system/bin/sh: cd: /sdcard: No such file or directory
2|root@generic_x86_64:/ # cd /storage/sdcard/
root@generic_x86_64:/storage/sdcard #
```

You cannot make an assumption about the path to external storage.

**Comment by tdrhq** [comment]
Hmm, okay. The hardcoding is also done on the python side which needs to be fixed along with this one.

**Comment by tdrhq** [comment]
Fixed in cdbebcc1a8f22803832bb13ed79dab13fb680220

If possible, please verify the fix and I'll have a new release out by EOW

**Comment by scottkennedy** [comment]
Yep, this solved it, thanks.
